src/main/java/edu/oregonstate/cs361/battleship/Main.java

```diff
@@ -182,137 +143,114 @@ public static String placeShipUpdated(Request req) {
-
-
-            if (stop == 1) {
-                for (movingPoint = movingPoint - 1; movingPoint < (computer_length_increase + 2); movingPoint++) {
-                    if (horizontal == 1) {
-                        cord.setAcross(movingPoint);
-                        cord.setDown(stoppedPoint);
-
-                    } else {
-                        cord.setAcross(stoppedPoint);
-                        cord.setDown(movingPoint);
-                    }
-
-
-                    //if ship lands on another ship then
-                    if (Hit(CAircraftCarrier.getStart(), CAircraftCarrier.getEnd(), cord) || Hit(CBattleship.getStart(), CBattleship.getEnd(), cord) || Hit(CSubmarine.getStart(), CSubmarine.getEnd(), cord) || Hit(CClipper.getStart(), CClipper.getEnd(), cord) || Hit(CDinghy.getStart(), CDinghy.getEnd(), cord)) {
-                        stop = 0;
-                    }
-                    /*
-                    if (Hit(CAircraftCarrier.getStart(), CAircraftCarrier.getEnd(), cord)) {
-                        stop = 0;
-                    } else if (Hit(CBattleship.getStart(), CBattleship.getEnd(), cord)) {
-                        stop = 0;
-                    } else if (Hit(CSubmarine.getStart(), CSubmarine.getEnd(), cord)) {
-                        stop = 0;
-                    } else if (Hit(CClipper.getStart(), CClipper.getEnd(), cord)) {
-                        stop = 0;
-                    } else if (Hit(CDinghy.getStart(), CDinghy.getEnd(), cord)) {
-                        stop = 0;
-                    }*/
+                    movingPoint = computer_y;
+                    stoppedPoint = computer_x;
 
+                } else {
+                    stop = 0;
                 }
 
-                if (stop == 1) {
-                    if (id.toLowerCase().contains("aircraftcarrier")) {
-                        CAircraftCarrier.setStart(computerStart.getAcross(), computerStart.getDown());
-                        CAircraftCarrier.setEnd(computerEnd.getAcross(), computerEnd.getDown());
-
-                    } else if (id.toLowerCase().contains("battleship")) {
-                        CBattleship.setStart(computerStart.getAcross(), computerStart.getDown());
-                        CBattleship.setEnd(computerEnd.getAcross(), computerEnd.getDown());
 
-                    } else if (id.toLowerCase().contains("submarine")) {
-                        CSubmarine.setStart(computerStart.getAcross(), computerStart.getDown());
-                        CSubmarine.setEnd(computerEnd.getAcross(), computerEnd.getDown());
-
-                    } else if (id.toLowerCase().contains("dinghy")) {
-                        CDinghy.setStart(computerStart.getAcross(), computerStart.getDown());
-                        CDinghy.setEnd(computerEnd.getAcross(), computerEnd.getDown());
+                if (stop == 1) {
+                    for (movingPoint = movingPoint - 1; movingPoint < (computer_length_increase + 2); movingPoint++) {
```

Comment:
computer length increase is the length on the ships from the starting location to the end.